src/kixi/paloma/enrich/events.clj

```diff
@@ -0,0 +1,33 @@
+(ns kixi.paloma.enrich.events
+  (:require [kixi.paloma.enrich.llpg :as llpg]))
+
+(defn data-source [acc event]
+  (keyword (:data-source event)))
+
+(defmulti rollup-event data-source)
```

Comment:
This should probably be named `merge-record`, but that can change in the next round of dev

Reply:
I've made a comment in my local copy.